iidm/iidm-impl/src/test/java/com/powsybl/iidm/network/impl/BusBreakerVoltageLevelTest.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.impl;
+
+import com.powsybl.iidm.network.*;
+import com.powsybl.math.graph.*;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author Florian Dupuy <florian.dupuy at rte-france.com>
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public class BusBreakerVoltageLevelTest {
+
+    private static VoltageLevel createVoltageLevel() {
+
+        Network network = Network.create("bus/breaker network", "test");
+
+        // The following code shows how to create the substation with a bus/breaker topology model.
+        Substation  substation = network.newSubstation()
+                .setId("susbtation")
+                .add();
+
+        // We create a voltage level vl1 which nominal voltage is 132.
+        VoltageLevel vl1 = substation.newVoltageLevel()
+                .setId("VL1")
+                .setTopologyKind(TopologyKind.BUS_BREAKER)
+                .setNominalV(132)
+                .add();
+
+        // Voltage level vl1 has two buses.
+        vl1.getBusBreakerView().newBus()
+                .setId("B1")
+                .add();
+        vl1.getBusBreakerView().newBus()
+                .setId("B2")
+                .add();
+
+        // We create a breaker between the two buses.
+        vl1.getBusBreakerView().newSwitch()
+                .setId("BR3")
+                .setBus1("B1")
+                .setBus2("B2")
+                .add();
+
+        // We create a generator connected through bus B1.
+        vl1.newGenerator()
+                .setId("GN")
+                .setBus("B1")
+                .setMinP(0)
+                .setMaxP(100)
+                .setTargetP(80)
+                .setTargetQ(50)
+                .setVoltageRegulatorOn(false)
+                .add();
+
+        // We create a load disconnected but which can be connected through B1.
+        vl1.newLoad()
+                .setId("LD")
+                .setConnectableBus("B1")
+                .setP0(40)
+                .setQ0(0)
+                .add();
+
+        return vl1;
+    }
+
+    @Test
+    public void testBusBreakerVoltageLevel() {
+        VoltageLevel vl = createVoltageLevel();
+        assertEquals(1000, BusBreakerVoltageLevel.NODE_LIMIT);
+    }
+
+    @Test
+    public void testBusBreakerVoltageLevelConfig() {
+        VoltageLevel vl = createVoltageLevel();
+        assertEquals(5, BusBreakerVoltageLevel.NODE_LIMIT);
```

Comment:
@flo-dup I am not really sure how I can test the configuration here. I was thinking it would be good to configure the node-limit and verify that the BusBreakerVoltageLevel is using the correct value.